integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -1050,4 +1065,76 @@ private static void checkCmdInLoopForDelete(String cmd, String matchStr, String
       }
     }
   }
+
+  // create yaml file with changed property
+  public static void createNewYamlFile(
+      String inputYamlFile, String generatedYamlFile, String oldString, String newString)
+      throws Exception {
+    logger.info("Creating new  " + generatedYamlFile);
+
+    Files.copy(
+        new File(inputYamlFile).toPath(),
+        Paths.get(generatedYamlFile),
+        StandardCopyOption.REPLACE_EXISTING);
+
+    // read each line in input domain file and replace with intended changed property
+    BufferedReader reader = new BufferedReader(new FileReader(generatedYamlFile));
+    String line = "";
+    StringBuffer changedLines = new StringBuffer();
+    boolean isLineChanged = false;
+    while ((line = reader.readLine()) != null) {
+      if (line.contains(oldString)) {
+        String changedLine = line.replace(line.substring(line.indexOf(oldString)), newString);
+        changedLines.append(changedLine).append("\n");
+        isLineChanged = true;
+      }
+
+      if (!isLineChanged) {
+        changedLines.append(line).append("\n");
+      }
+      isLineChanged = false;
+    }
+    reader.close();
+    // writing to the file
+    Files.write(Paths.get(generatedYamlFile), changedLines.toString().getBytes());
+    logger.info("Done - generate the new yaml file ");
+  }
+
+  public static void copyFile(String fromFile, String toFile) throws Exception {
+    logger.info("Copying file from  " + fromFile + " to " + toFile);
+
+    Files.copy(new File(fromFile).toPath(), Paths.get(toFile), StandardCopyOption.REPLACE_EXISTING);
+  }
+
+  public static void dockerTagImage(String sourceImage, String targetImage) throws Exception {
+    logger.info("Tagging souceImage:  " + sourceImage + "  to " + targetImage);
+    String dockerCmd = "docker tag " + sourceImage + " " + targetImage;
+    logger.info("Executing cmd " + dockerCmd);
+
+    ExecResult result = ExecCommand.exec(dockerCmd);
```

Comment:
See if you can replace the below lines with TestUtils.exec()

Reply:
Probably not. TestUtils.ExecResult exec(String cmd) and I need 2 parameters

Reply:
Took another look. Gotcha :) Thanks!